### ISSUE TITLE
[Feature Request] Format indentation

### ISSUE THREAD

**Issue by sakgoyal** [zenstackhq/zenstack#885]
**Is your feature request related to a problem? Please describe.**
Can you add a way for the format to align all items in a model so its easier to read?
current:
```
model User {
	id ...
    somethinglong ...
	blah ...
}
```
expected:
```
model User {
	id             ...
    somethinglong  ...
	blah           ...
}

```

**Comment by ymc9** [comment]
Thanks for filing this @sakgoyal .

@jiashengguo I remember we had a related feature request for it but I couldn't find it ...

**Comment by nopain1210** [comment]
We've written a bash script to comment out all the utility attributes of zenstack, then format it using `prisma format`, and finally remove the commented-out sections. We hope this feature will soon be available in the next version.

**Comment by sakgoyal** [comment]
I think that might complain if you use certain features like abstract models since that is not native to Prisma. Just commenting out attributes won't work as far as I know

**Comment by ymc9** [comment]
Thanks for the comments, guys. We're working on this feature now!

**Comment by ymc9** [comment]
Fixed in 2.0.0-beta.13
